spread_tests/lib/prepare-all.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Simulate a WiFi radio network interface
+#modprobe mac80211_hwsim radios=2
+
+# We don't have to build a snap when we should use one from a
+# channel
+if [ -n "$SNAP_CHANNEL" ] ; then
+	exit 0
+fi
+
+# If there is a wifi-connect snap prebuilt for us, lets take
+# that one to speed things up.
+if [ -e ${PROJECT_PATH}/${SNAP_NAME}_*_${SNAP_ARCH}.snap ] ; then
+	exit 0
+fi
+
+# Search for updates
+#snap refresh
+
+# Setup classic snap and build the wifi-connect snap in there
+snap install --devmode --beta classic
+
+cat <<-EOF > /home/test/build-snap.sh
+#!/bin/sh
+set -ex
+apt update
+apt install -y --force-yes snapcraft
```

Comment:
run-tests fails for me with several of these:
go get -t -d ./github.com/CanonicalLtd/UCWifiConnect/...
go: missing Git command. See https://golang.org/s/gogetcmd
package github.com/presotto/go-mdns-sd: exec: "git": executable file not found in $PATH
go: missing Git command. See https://golang.org/s/gogetcmd

Until I add the following line here to install git in the backend:
apt install -y --force-yes git